services/notify/test/api_test.js

```diff
@@ -4,8 +4,12 @@ const helper = require('./helper');
 const testing = require('taskcluster-lib-testing');
 
 helper.secrets.mockSuite(testing.suiteName(), ['db', 'aws'], function(mock, skipping) {
+  // Temporary workaround for
+  //   https://github.com/taskcluster/taskcluster/issues/3131
+  if (mock) {
+    return;
+  }
```

Comment:
No need for this condition. I think as long as TEST_DB_URL is defined as an environment variable, the services will use the local db to run its tests.

Reply:
My experience when running tests locally was that the mocks were still run, but I can try again tomorrow. Once I added this, the tests just ran against the real database.

Reply:
I just tried locally and I can confirm that mocks are not run when TEST_DB_URL is specified.

Reply:
I tried removing, but then I get CI failures, so I've moved it into a separate third commit. I think when we tackle #3131 we can also revert this third commit that just takes care of skipping fake db tests.

Reply:
Sounds fine..

Reply:
I've rebased now, and it looks like the third commit is no longer needed after all, so maybe I was missing something before. In any case I'll drop the third commit now...